website/.vitepress/components/Home.vue

```diff
@@ -19,7 +19,12 @@
             </Teleport>
             <Teleport to="#export-diagram">
                 <VPFlyout icon="vpi-download" label="Download diagram" class="download-flyout">
-                    <button class="menu-button" :disabled="diagram == undefined" @click="downloadSVG">SVG</button>
+                    <button class="menu-button" :disabled="diagram == undefined" @click="downloadSVG(false)">
+                        SVG
+                    </button>
+                    <button class="menu-button" :disabled="diagram == undefined" @click="downloadSVG(true)">
+                        SVG (text as path)
```

Comment:
```suggestion
                        SVG (text as image)
```
not 100% correct, but probably easier to understand for users

Reply:
I would prefer path
that is also what diagrams.net used previously (they now completely removed the option)
also image really sounds like raster image, what it isn't

Reply:
I disagree on that:
You must first have enough technical knowledge to know that an SVG consists of `paths` which not every user will know.
The concept of an `image` is however well-understood, even if more technical users might be a little confused what that means under the hood.

Anyway, that's nothing that should block this PR